zebra-state/src/service/finalized_state/disk_format.rs

```diff
@@ -163,6 +163,14 @@ impl IntoDisk for sapling::Nullifier {
     }
 }
 
+impl IntoDisk for orchard::Nullifier {
+    type Bytes = [u8; 32];
+
+    fn as_bytes(&self) -> Self::Bytes {
+        <[u8; 32]>::from(self.0)
```

Comment:
We have `impl From<orchard::Nullifier> for [u8; 32]` for this use case, and unless we have to, we don't want to validate the abstract/API boundary by reaching inside newtypes to get at their data, especially when we control/author those types.

```suggestion
        self.into()
```

Reply:
These `IntoDisk` impls for the other nullifiers are ok given that those types wrap just bytes, with `orchard::Nullifier` we can/should leverage the [u8; 32] conversion to avoid coupling to the guts too much

Reply:
Thanks, I applied but needed an extra line for deref. https://github.com/ZcashFoundation/zebra/pull/2185/commits/ae61dea825116080a452c96dbf2e8f0979daf433